src/ssr-carousel.vue

```diff
@@ -166,6 +170,9 @@ export default
 .ssr-carousel
 	touch-action pan-y
 
+	// Internal logic expects ltr layout
+	direction ltr
+
```

Comment:
@azerozvn I realized one possible issue with this change. I'm setting the direction of the carousel wrapper to `ltr` so that my reversing of the order works as intended.  However, this means that the slides will inherit this setting.

Like notice how the slide titles here are `ltr`:

![image](https://github.com/BKWLD/vue-ssr-carousel/assets/77567/8bb2eb52-ab8f-4ffe-8c4a-9c94cedee56f)

I want to release a patch fix that sets the carousel track contents to `direction: rtl` when the `rtl` prop of the container is set.  See any issue with that?

Reply:
Alright @azerozvn this change was released as v2.4.1 if you wanna upgrade to it.  See #138.